tests/RobotFramework/tests/cumulocity/restart/restart_device_child.robot

```diff
@@ -0,0 +1,64 @@
+*** Settings ***
+Resource            ../../../resources/common.resource
+Library             Cumulocity
+Library             ThinEdgeIO
+
+Suite Setup         Custom Setup
+Test Teardown       Get Logs
+
+Test Tags           theme:c8y    theme:troubleshooting    theme:childdevices
+
+
+*** Test Cases ***
+Support restarting the device
+    Cumulocity.Should Contain Supported Operations    c8y_Restart
```

Comment:
@reubenmiller Where is this check defined?

Reply:
The assertion checks if the `c8y_Restart` operation type has been declared in Cumulocity IoT REST API.

The capabilities are registered fine on the tedge broker, however they are definitely not translated to cloud messages. I believe this functionality is missing as the c8y_LogFileRequest operation is also not sent to the cloud.